examples/hubert/dataset/hubert_dataset.py

```diff
@@ -101,7 +105,10 @@ def _get_buckets(self, lengths: List[int], num_buckets: int, min_len: int, max_l
         buckets = {k: v for k, v in sorted(buckets.items())}
         return buckets
 
-    def _update_iter_list(self) -> None:
+    def _update_iter_list(self, generator=None) -> None:
```

Comment:
do we need kwarg `generator`?

Reply:
I was thinking shuffling `BucketizeBatchSampler` with the same generator guarantees each GPU gets the same `iter_list`, after second thought, `BucketizeBatchSampler` is already initialized when it is passed to `DistributedBatchSampler`. So we don't need to worry about it. Will remove it, thanks.